schema/definitions/0.8.0/schema/fmu_meta.json

```diff
@@ -1,41 +1,43 @@
 {
   "$contractual": [
+    "access",
     "class",
-    "source",
-    "version",
-    "tracklog",
-    "data.format",
-    "data.name",
-    "data.stratigraphic",
     "data.alias",
-    "data.stratigraphic_alias",
-    "data.offset",
+    "data.bbox",
     "data.content",
-    "data.tagname",
-    "data.vertical_domain",
+    "data.format",
     "data.grid_model",
-    "data.bbox",
-    "data.time",
-    "data.is_prediction",
     "data.is_observation",
+    "data.is_prediction",
+    "data.name",
+    "data.offset",
     "data.seismic.attribute",
     "data.spec.columns",
-    "access",
-    "masterdata",
-    "fmu.model",
-    "fmu.workflow",
+    "data.stratigraphic",
+    "data.stratigraphic_alias",
+    "data.tagname",
+    "data.time",
+    "data.vertical_domain",
+    "file.checksum_md5",
+    "file.relative_path",
+    "file.size_bytes",
+    "fmu.aggregation.operation",
+    "fmu.aggregation.realization_ids",
     "fmu.case",
+    "fmu.context.stage",
     "fmu.iteration.name",
     "fmu.iteration.uuid",
-    "fmu.realization.name",
+    "fmu.model",
```

Comment:
This is just an alphabetical sort on this list. The only items that changed are tracklog.